### ISSUE TITLE
Staging Bugs | Anchor Tags

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#23920]
Per review of staging link, the following fixes are being requested:

- https://staging.va.gov/coronavirus-veteran-frequently-asked-questions-esp/#como-puedo-seguir-informado-e-27876 

- replace with English version of Corona FAQ page ( https://staging.va.gov/coronavirus-veteran-frequently-asked-questions/ )
-

**Comment by SKasimow** [comment]
@youngfreezyVA @jenniferlee-dsva @rtwell Hi All - I've consolidated our staging feedback in this ticket; let me know if I missed anything or if anything new is uncovered and I can add.

@youngfreezyVA - can you point this?

**Comment by SKasimow** [comment]
@youngfreezyVA for item 

re: 2 - still seeing the icon/text revert in stages. Can the icon and text revert simultaneously?

re: 4 still not working as described above

**Comment by jenniferlee-dsva** [comment]
@SKasimow - your consolidation of bugs reported on Slack LGTM. Awesome job - thank you! 

I haven't gone back to revalidate. Will do so again after the go-ahead from you and @youngfreezyVA . BTW, I'm okay checking Monday, as I'm trying to prep for a meeting tomorrow. )

**Comment by SKasimow** [comment]
@jenniferlee-dsva sure - np. I didn't realize that @youngfreezyVA updates had not been merged yet which is why I was not seeing a change on above ;)

**Comment by jenniferlee-dsva** [comment]
I was able to validate all of the above except for #s 3 and 7 because the non-accordion h3 contents have disappeared on the FAQs page. 

3. Chrome Issue with "Link copied" behavior: In Chrome/ mobile view: same behavior as above: the icon doesn't ever go back to default blue state after the 10 seconds. It just stays black until you click on the next icon.

7. Text Placement: in this example, “Link copied” should be able to go to the right, but it isn’t:

**Comment by SKasimow** [comment]
@youngfreezyVA

new bug: when using safari, after 10 seconds the icon turns from black background to white but still has a yellow outline

**Comment by youngfreezyVA** [comment]
i’m not sure if that’s a bug. The yellow outline indicates focus on the element, it stays on after it was clicked. You would have to tab or click onto something else for the yellow outline to go away. lmk if i'm mis understanding, a video of the bug would be helpful.

**Comment by rtwell** [comment]
yeah i do not believe that's a bug—focus should stay with the element until the user has moved it

**Comment by SKasimow** [comment]
The yellow outline DOES go away if I click elsewhere on page

**Comment by SKasimow** [comment]
@youngfreezyVA Danielle updated the Corona FAQ page to include a question that will live outside of an accordion so we have that additional use case for the pilot.

Is there something you need to do to get the share-link icon back onto staging page for this non-accordion QA? I know you said for accordion QA the icon is part fo the template so would appear auto-magically but not sure about QA's outside accordions?

https://staging.va.gov/coronavirus-veteran-frequently-asked-questions/

**Comment by youngfreezyVA** [comment]
as long as the non accordion is a template that was previously compatible
with sharable links, we are good to go.

If you are referring to the following question: `Can I get a COVID-19
vaccine at VA?
`
that is a new template.... we need to ensure backwards compatibility when
updating the layout.  I've previously shared (on slack) the templates that
I updated as per the sharable links requirements.

That is the paragraphs/wysiwyg template, which wasn't in scope for previous
requirements.

[image: GovernmentCIO] <http://www.governmentcio.com/>
Fareez Ahmed
Sr Software Engineer
| *mobile* (202) 677-0160
| *e.* ***@***.***
*Government*CIO
website <https://www.governmentcio.com/> | media
<https://governmentciomedia.com/> | services
<https://www.governmentcio.com/services> | jobs
<https://careers-governmentcio.icims.com/> | map
***@***.***,-77.0151008,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7b787cdfdb85f:0x1e5a93a8a2b38b03!8m2!3d38.8927345!4d-77.0129121>
[image: Twitter] <https://twitter.com/GovernmentCIO> [image: Facebook]
<https://www.facebook.com/GovernmentCIOMedia> [image: LinkedIn]
<https://www.linkedin.com/company/governmentcio> [image: YouTube]
<https://www.youtube.com/user/GovernmentCIO/videos> [image: Instagram]
<https://www.instagram.com/gciotalent>
*Confidentiality Note: *This email may contain confidential and/or private
information.
If you received this email in error please delete and notify sender.

**Comment by youngfreezyVA** [comment]
<img width="358" alt="Screen Shot 2021-04-23 at 9 16 16 AM" src="https://user-images.githubusercontent.com/46791771/117330883-fec3f480-ae4a-11eb-9265-d8b3d098dfaa.png">

**Comment by SKasimow** [comment]
Hi @DanielleThierryUSDSVA - per above comment ^ from Fareez (and previous slack thread here: https://dsva.slack.com/archives/C01GCADCMQT/p1619178556145800)


...the share-link icon is only picked up when we use one of the original templates that Fareez coded for.
The QA you took out of the accordion yesterday is now in a "new" template, per Fareez. Would it be possible for you to re-do the QA into one of the original templates that were not in part of an accordion?

(Fareez - jump in if I am not capturing this correctly!)

cc @jenniferlee-dsva

**Comment by youngfreezyVA** [comment]
yes that’s correct! alternatively, we may need to add the share link to an additional template, but I think adding to the paragraphs/wysiwg template would lead to a lot of unintended side effects and the share link appearing in places we may not want it (for example, inside the additional info label)
